openmmtools/utils/equilibration.py

```diff
@@ -0,0 +1,164 @@
+"""
+Utility functions useful for equilibration.
+"""
+
+import logging
+from openmm import unit, OpenMMException
+
+# Set up logger
+_logger = logging.getLogger(__name__)
+
+
+def gentle_equilibration(topology, positions, system, stages, filename, platform_name='CUDA', save_box_vectors=True):
```

Comment:
Should we rename the function to something that starts with a verb? Suggestions: `run_gentle_equilibration` or `equilibrate`

Reply:
I agree, that's a better name. Thanks!